.azurepipelines/Ubuntu-GCC5.yml

```diff
@@ -10,9 +10,9 @@ resources:
   repositories:
     - repository: mu_devops
       type: github
-      endpoint: microsoft
-      name: microsoft/mu_devops
-      ref: main
+      endpoint: cfernald
+      name: cfernald/mu_devops
```

Comment:
Should this be migrated to the main mu_devops repo, not your fork (if I'm reading this correctly)?

Reply:
Correct, will update and publish once the first PR is in